libraries/chain/include/eosio/chain/block.hpp

```diff
@@ -129,3 +142,20 @@ FC_REFLECT_DERIVED(eosio::chain::transaction_receipt, (eosio::chain::transaction
 FC_REFLECT(eosio::chain::additional_block_signatures_extension, (signatures));
 FC_REFLECT(eosio::chain::quorum_certificate_extension, (qc));
 FC_REFLECT_DERIVED(eosio::chain::signed_block, (eosio::chain::signed_block_header), (transactions)(block_extensions) )
+
```

Comment:
Why not put this code some where in fc?

Reply:
Not sure I follow. `signed_block` is defined in `libraries/chain` which depends on `libraries/libfc`.
I did try to move the forward declare out of `fc`, but couldn't get it to work without the `fc` forward declare.

Reply:
I meant `unpack(Stream& s, eosio::chain::signed_block& v)` is defined in `raw_fwd.hpp`, why not put the implementation there. I think about it more. It is also OK to remain here.